Automattic/wp-calypso
# Patterns Library: Start tracking calypso_pattern_library_view

see: pcYYuD-1hV-p2#calypso_pattern_library_view

This set of changes extend the `<PatternsPageViewTracker/>` component such that:

- `calypso_pattern_library_search` is no longer being tracked.
- `calypso_pattern_library_view`  is now being tracked

## Testing
- Navigate to any page in the `/patterns/*` route
- confirm the aforementioned events
